addons/a11y/src/components/A11YPanel.test.tsx

```diff
@@ -129,11 +129,14 @@ describe('A11YPanel', () => {
     const { getByText } = render(<ThemedA11YPanel />);
     const useChannelArgs = mockedApi.useChannel.mock.calls[0][0];
     act(() => useChannelArgs[EVENTS.RESULT](axeResult));
-    await waitFor(() => {
-      expect(getByText(/Tests completed/)).toBeTruthy();
-      expect(getByText(/Violations/)).toBeTruthy();
-      expect(getByText(/Passes/)).toBeTruthy();
-      expect(getByText(/Incomplete/)).toBeTruthy();
-    });
+    await waitFor(
+      () => {
+        expect(getByText(/Tests completed/)).toBeTruthy();
+        expect(getByText(/Violations/)).toBeTruthy();
+        expect(getByText(/Passes/)).toBeTruthy();
+        expect(getByText(/Incomplete/)).toBeTruthy();
+      },
+      { timeout: 2000 }
```

Comment:
This timeout didn't actually seem to help, this test still fails intermittently.  Would love if someone who knows more about the a11y panel could take a look at why.  But even so, I think it might be good to merge this even knowing this test is flakey, just so we can get some windows tests running.